src/WebSocketServer/WebSocketServerPlugin.cs

```diff
@@ -316,8 +316,8 @@ private void StartWebSocketServer()
                         {
                             if (err != SslPolicyErrors.None)
                                 return false;
```

Comment:
If chain is null ||

Reply:
Specifies a callback for additional client certificate validation that will be invoked during authentication. This will be ignored if [AllowAnyClientCertificate()](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.server.kestrel.https.httpsconnectionadapteroptions.allowanyclientcertificate?view=aspnetcore-7.0#microsoft-aspnetcore-server-kestrel-https-httpsconnectionadapteroptions-allowanyclientcertificate) is called after this callback is set.

Reply:
So no need to check `null`. `null` isnt passed.